examples/accordion/css/accordion.css

```diff
@@ -20,20 +30,33 @@
     position: relative;
     text-align: left;
     width: 100%;
+    outline: none;
 }
 
-.Accordion dt:first-child .Accordion-trigger {
-    border-radius: .3em .3em 0 0;
+.Accordion h3 {
+    margin: 0;
+    padding: 0;
+}
+
+.Accordion *:first-child .Accordion-trigger {
+    border-radius: 5px 5px 0 0;
 }
 
 .Accordion-trigger:focus,
 .Accordion-trigger:hover {
-    background: hsl(0, 0%, 93%);
+    background: hsl(216, 94%, 94%);
 }
 
 .Accordion-title {
-  display: block; /* For Edge bug https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8295099/ */
   pointer-events: none;
+  border: transparent 2px solid;
+  border-radius: 5px;
+  padding: 0.25em;
+  outline: none;
+}
+
+.Accordion-trigger:focus .Accordion-title {
+    border-color: hsl(216, 94%, 73%);
```

Comment:
The `.Accordion-trigger` already has a visual change on focus, so I'm not entirely clear what purpose this serves. It also gives the visual impression that only the `.Accordion-title` is focused, or that the `.Accordion-title` is the clickable area, neither of which is true.